Libraries/Image/RCTImageLoader.m

```diff
@@ -377,10 +371,24 @@ - (RCTImageLoaderCancellationBlock)loadImageOrDataWithURLRequest:(NSURLRequest *
                                                    userInfo:nil], nil);
           return;
         }
-      }
 
-      // Call handler
-      completionHandler(nil, data);
+        NSString *responseDate = ((NSHTTPURLResponse *)response).allHeaderFields[@"Date"];
```

Comment:
@mmmulani But, before editing, then how can we know that response has cached or not?
In that case, what about ignore checking cache?